server/index.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+
+import db from '../sequelize/models/index';
+const { sequelize } = db;
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+const port = process.env.PORT || 3000;
+
+sequelize.sync().then(() => {
+    app.listen(port, () => {
+        console.log('Database succesfully connected and server listening on port');
+    });
+});
+
+app.get('**', (req, res) => {
+  res.status(400).send({
+    status: 400,
```

Comment:
You are replying to every request with this handler. Even though it is just to test the status should be `200`  not `400`.